src/Settings.ts

```diff
@@ -82,6 +82,16 @@ export namespace Settings {
         }
     }
 
+    export namespace SettingsFile {

```

Comment:
Be consistent with the name. Instead of a namespace, simply declare `settingFile` as a function.